tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/Trusts/ContactsModelTests.cs

```diff
@@ -9,30 +11,29 @@ namespace DfE.FindInformationAcademiesTrusts.UnitTests.Pages.Trusts;
 public class ContactsModelTests
 {
     private readonly ContactsModel _sut;
-    private readonly Mock<ITrustProvider> _mockTrustProvider;
-    private readonly Trust _dummyTrustWithGovernors;
-    private readonly Trust _dummyTrustWithNoGovernors;
+    private readonly Mock<ITrustProvider> _mockTrustProvider = new();
     private const string PresentChairOfTrustees = "Present ChairOfTrustees";
     private const string PresentAccountingOfficer = "Present AccountingOfficer";
     private const string PresentChiefFinancialOfficer = "Present ChiefFinancialOfficer";
     private const string ChairOfTrustees = "Chair Of Trustees";
     private const string AccountingOfficer = "Accounting Officer";
     private const string ChiefFinancialOfficer = "Chief Financial Officer";
-    private static DateTime _currentGovernorDate;
-    private static DateTime _pastGovernorDate;
+    private static readonly DateTime CurrentGovernorDate = DateTime.Today;
```

Comment:
Is it worth using our DateTimeProvider for consistency

Reply:
These dates are being used by unit tests for flavour, there is no logic based on these dates and this is a refactor so will not fix